adapter-pinecone/pom.xml

```diff
@@ -0,0 +1,66 @@
+    <!--
+  ~ Copyright (c) 2022-2023 nosqlbench
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<modelVersion>4.0.0</modelVersion>
+
+<artifactId>adapter-pinecone</artifactId>
+<packaging>jar</packaging>
+
+<parent>
+    <artifactId>mvn-defaults</artifactId>
+    <groupId>io.nosqlbench</groupId>
+    <version>${revision}</version>
+    <relativePath>../mvn-defaults</relativePath>
+</parent>
+
+<name>${project.artifactId}</name>
+<description>
+    An nosqlbench ActivityType (AT) driver module;
+    Provides basic formatting and output to stdout or files.
```

Comment:
*nit - we may need to adjust this to match pinecone interaction.

Reply:
+1

Reply:
Fixed in latest push